monitoring/opencensus/README.rst.in

```diff
@@ -0,0 +1,21 @@
+# This file is used to generate README.rst
```

Comment:
Thanks for adding this! Please also trigger the README generation:

1. Install nox. `pip install nox`.
2. Temporarily copy the noxfile from the root of the repo into this directory. `cp ../../noxfile-template.py noxfile.py`
3. `nox -s readmegen` (this should create a README.rst)
4. Delete the `noxfile.py`.

Reply:
I'm trying to do that now - running `nox -s readmegen` results in this:

```
$ nox -s readmegen
No user noxfile_config found: detail: No module named 'noxfile_config'
nox > Running session readmegen(path=PosixPath('README.rst.in'))
nox > Creating virtual environment (virtualenv) using python3.9 in .nox/readmegen-path-posixpath-readme-rst-in
nox > python -m pip install jinja2 pyyaml
nox > python -m pip install -r requirements.txt
nox > python /Users/ygrinshteyn/src/forks/python-docs-samples/scripts/readme-gen/readme_gen.py README.rst.in
/Users/ygrinshteyn/src/forks/python-docs-samples/scripts/readme-gen/readme_gen.py:54: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  config = yaml.load(f)
Dropping 1 non-positive bucket boundaries
 * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
```

It just seems to hang here forever...

Reply:
This is what happens when I interrupt it:

```
Traceback (most recent call last):
  File "/Users/ygrinshteyn/src/forks/python-docs-samples/scripts/readme-gen/readme_gen.py", line 66, in <module>
    main()
  File "/Users/ygrinshteyn/src/forks/python-docs-samples/scripts/readme-gen/readme_gen.py", line 59, in main
    output = README_TMPL.render(config)
  File "/Users/ygrinshteyn/src/forks/python-docs-samples/monitoring/opencensus/.nox/readmegen-path-posixpath-readme-rst-in/lib/python3.9/site-packages/jinja2/environment.py", line 1088, in render
    return concat(self.root_render_func(self.new_context(vars)))
  File "/Users/ygrinshteyn/src/forks/python-docs-samples/scripts/readme-gen/templates/README.tmpl.rst", line 98, in root
  File "/Users/ygrinshteyn/src/forks/python-docs-samples/monitoring/opencensus/.nox/readmegen-path-posixpath-readme-rst-in/lib/python3.9/site-packages/jinja2/runtime.py", line 290, in call
    return __obj(*args, **kwargs)
  File "/Users/ygrinshteyn/src/forks/python-docs-samples/scripts/readme-gen/readme_gen.py", line 37, in get_help
    return subprocess.check_output(['python', file, '--help']).decode()
  File "/Users/ygrinshteyn/.pyenv/versions/3.9.1/lib/python3.9/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/Users/ygrinshteyn/.pyenv/versions/3.9.1/lib/python3.9/subprocess.py", line 503, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
  File "/Users/ygrinshteyn/.pyenv/versions/3.9.1/lib/python3.9/subprocess.py", line 1117, in communicate
    stdout = self.stdout.read()
KeyboardInterrupt
nox > Interrupted...
nox > Session readmegen(path=PosixPath('README.rst.in')) interrupted.
```

Reply:
Huh strange. 🤔  We're kind of moving away from the generated README.rsts in their current form. Would you mind re-working this to be a standalone README.md? See https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/appengine/standard for an example.

Reply:
Done.